packages/salesforcedx-vscode-core/src/commands/forceSourceDeployManifest.ts

```diff
@@ -34,12 +34,16 @@ export class LibrarySourceDeployManifestExecutor extends DeployExecutor<
     response: ContinueResponse<string>
   ): Promise<ComponentSet> {
     const packageDirs = await SfdxPackageDirectories.getPackageDirectoryPaths();
-    return ComponentSet.fromManifest({
+    const rootWorkspacePath = workspaceUtils.getRootWorkspacePath();
+    const resolveSourcePaths = packageDirs.map(packageDir => join(rootWorkspacePath, packageDir));
+
+    const componentSet = await ComponentSet.fromManifest({
       manifestPath: response.data,
-      resolveSourcePaths: packageDirs.map(dir =>
-        join(workspaceUtils.getRootWorkspacePath(), dir)
-      )
+      resolveSourcePaths
+      // forceAddWildcards jab ?
```

Comment:
forceSourceRetrieveManifest passes forceAddWildcards into `ComponentSet.fromManifest()` but forceSourceDeployManifest doesn't.  Is it missing or was this intentional?

If intentional, I think:
`   forceAddWildcards: undefined`
...should be added so it's more clear.

Reply:
`forceAddWildcards` is:
```
forceAddWildcards?: boolean;
```
...and so this is an optional parameter and can be undefined.

Reply:
Update: I checked with the CLI team and it appears this was intentional.  I'm going to add 'forceAddWildcards: undefined' here to make it move obvious & clear.